arduino-ide-extension/src/browser/contributions/startup-tasks-executor.ts

```diff
@@ -0,0 +1,62 @@
+import { DisposableCollection } from '@theia/core/lib/common/disposable';
+import {
+  inject,
+  injectable,
+  postConstruct,
+} from '@theia/core/shared/inversify';
+import { hasStartupTasks } from '../../electron-common/startup-task';
+import { AppService } from '../app-service';
+import { Contribution } from './contribution';
+
+@injectable()
+export class StartupTasksExecutor extends Contribution {
+  @inject(AppService)
+  private readonly appService: AppService;
+
+  private readonly toDispose = new DisposableCollection();
+
+  @postConstruct()
+  protected override init(): void {
+    this.toDispose.push(
+      this.appService.registerStartupTasksHandler((args) =>
+        this.handleStartupTasks(args)
+      )
+    );
+  }
+
+  onStop(): void {
+    this.toDispose.dispose();
+  }
+
+  private async handleStartupTasks(args: unknown): Promise<void> {
```

Comment:
`args` can be a type of `StartupTasks`

Reply:
https://github.com/arduino/arduino-ide/pull/2027/commits/241baca40be0ed10b4d872c65f96d9034190f358